### PR TITLE
Create bukumark.json

### PR DESCRIPTION
**Pull request by w3teal**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

### https://bukumark.pages.dev/

### CODE REVIEW

File: domains/bukumark.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Bookmark manager that keep you simple with beautiful UI",
+    "repo": "https://github.com/LIGMATV/Bukumark",
+    "owner": {
+        "username": "LIGMATV",
+        "email": "ligmatv.id@gmail.com"
+    },
+    "record": {
+        "CNAME": "ligmatv.github.io/bukumark"
```
Comment (21Z) [inline domains/bukumark.json:9]: `CNAME` can't contain `/`

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/bukumark.json (hunk 1 above)
Comment (LIGMATV) [reply domains/bukumark.json:9]: Okey 👍🏻

File: domains/bukumark.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Bookmark manager that keep you simple with beautiful UI",
+    "repo": "https://github.com/LIGMATV/Bukumark",
+    "owner": {
+        "username": "LIGMATV",
+        "email": "ligmatv.id@gmail.com"
+    },
+    "record": {
+        "URL": "ligmatv.github.io/bukumark"
```
Comment (Stef-00012) [inline domains/bukumark.json:9]: missing protocol (`https://`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by LIGMATV** [comment]
... I thought i will use Cloudflare Pages instead this.

**Review by 21Z — APPROVED** [review]

**Pull request closed without merging**